sdk/Lusid.Sdk.Tests/tutorials/Ibor/ReferencePorfolios.cs

```diff
@@ -59,12 +60,35 @@ public void CreateReferencePortfolio()
             //  Confirm that the portfolio was successfully created.  Any failures will result in
             //  a ApiException being thrown which contain the relevant response code and error message
             Assert.That(ReferencePortfolio.Id.Code, Is.EqualTo(request.Code));
+
+            // Delete the portfolio once the test is complete
+            var _delete_portfolio_result = _apiFactory.Api<IPortfoliosApi>().DeletePortfolio(TutorialScope, F39PorfolioCode);
```

Comment:
maybe assert on the result, or remove the var

drop the `_` from the var name (use C# naming convention, its not private)

Reply:
either, but I think it might be good to assert it was successfully deleted